src/components/Scheme/ContributionRewardExtRewarders/Competition/CreateProposal.tsx

```diff
@@ -0,0 +1,626 @@
+                    name="votingStartDate"
+                    type="date"
+                    className={touched.startTime && errors.startTime ? css.error : null}
+                  />
+                  <div className={css.timeSelect}>
+                    <Field
+                      id="votingStartTimeInput"
+                      name="votingStartTime"
+                      type="time"
+                      component={SelectField}
+                      className={css.timeSelect}
+                      options={timeSlots}
+                    />
+                  </div>
+                </div>
+
+                <div className={css.date}>
+                  <img src="/assets/images/Icon/down.svg" className={css.downV}/>
+                  <label htmlFor="compEndDate">
+                    Competition end time
+                    <ErrorMessage name="compEndDate">{(msg) => <span className={css.errorMessage}>{msg}</span>}</ErrorMessage>
+                  </label>
+                  <Field
+                    id="compEndDateInput"
+                    name="compEndDate"
+                    type="date"
+                    className={touched.endDate && errors.endDate ? css.error : null}
+                  />
+                  <div className={css.timeSelect}>
+                    <Field
+                      id="compEndTimeInput"
+                      name="compEndTime"
+                      type="time"
+                      component={SelectField}
+                      className={css.timeSelect}
+                      options={timeSlots}
+                    />
+                  </div>
+                </div>
+
+                {(touched.ethReward || touched.externalTokenReward || touched.reputationReward || touched.nativeTokenReward)
+                    && touched.reputationReward && errors.rewards &&
+                <span className={css.errorMessage + " " + css.someReward}><br/> {errors.rewards}</span>
+                }
+              </div>
+              <div className={css.createProposalActions}>
+                <button className={css.exitProposalCreation} type="button" onClick={handleClose}>
+                  Cancel
+                </button>
+                <TrainingTooltip overlay="Once the proposal is submitted it cannot be edited or deleted" placement="top">
+                  <button className={css.submitProposal} type="submit" disabled={isSubmitting}>
+                  Submit proposal
+                  </button>
+                </TrainingTooltip>
+              </div>
+            </Form>
+          }
+        />
+      </div>
+    );
+  }
+}
+
+const SubscribedCreateContributionRewardExProposal = withSubscription({
+  wrappedComponent: CreateProposal,
+  checkForUpdate: ["daoAvatarAddress"],
+  createObservable: (props: IExternalProps) => {
+    const arc = getArc(); // TODO: maybe we pass in the arc context from withSubscription instead of creating one every time?
```

Comment:
`getArc` is not creating a new context but is returin "the" context, so we're already good

Reply:
please ignore that stupid comment that wasn't mine.  I'll go ahead and find all of them in the app and delete them